tshirts.c

```diff
@@ -17,6 +17,13 @@ int main() {
     assert(size(37) == 'S');
     assert(size(40) == 'M');
     assert(size(43) == 'L');
+    
+    //Added additional test cases
+    
+    assert(size(38) == 'S');
+    assert(size(38) == 'S');
+    assert(size(42) == 'M');
+    assert(size(42) == 'L');

```

Comment:
Is it possible for the method size() to return M and L subsequently for input value 42 ?